Atomic/diff.py

```diff
@@ -363,14 +420,29 @@ class DiffFS(object):
     """
     Primary class for doing a diff on two docker objects
     """
-    def __init__(self, chroot_left, chroot_right):
-        self.compare = dircmp(chroot_left, chroot_right)
+    def __init__(self, image_list):
+        #self.compare = dircmp(chroot_left, chroot_right)
+        self.img_left, self.img_right = (x for x in image_list)
         self.left = []
         self.right = []
         self.common_diff = []
-        self.chroot_left = chroot_left
-        self.chroot_right = chroot_right
-        self.delta(self.compare)
+        self.chroot_left = [] #chroot_left
+        self.chroot_right =[] # chroot_right
+        self.parse_mtree_json()
+
+    def parse_mtree_json(self):
+        def extra(_result): #pylint: disable=unused-variable
+            self.right.append(_result['path'])
+
+        def missing(_result): #pylint: disable=unused-variable
+            self.left.append(_result['path'])
+
+        def modified(_result): #pylint: disable=unused-variable
```

Comment:
Because of the use of eval below, pylint doesnt recognize they are used.

Leaving until better solution.